pkg/db/migrations/001_job.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TYPE job_result AS ENUM ('passed', 'failed', 'aborted');
```

Comment:
We should also have 'running' as an option, and have new rows default to that. I could see value in that for reporting or alerting.

Reply:
In my head, I was thinking we'd only be creating rows here at the end of a job, so we'd already know the pass/fail state. I guess we could create the record at the beginning, but I'm curious whether we'd gain much other than complexity. Thoughts?

Reply:
I was mentally thinking a 1:1 mapping between Prow job state and DB job state. For simplicity's sake now, I'm cool with keeping it to those 3 / end-of-job. 

Later on if we want that type of data, the change won't be too difficult.